airbyte-integrations/connectors/source-gitlab/source_gitlab/schemas/issues.json

```diff
@@ -98,15 +98,31 @@
     "service_desk_reply_to": {
       "type": ["null", "string"]
     },
+    "author": {
+      "type": ["null", "object"],
+      "additionalProperties": true
+    },
     "author_id": {
       "type": ["null", "integer"]
     },
+    "assignee": {
+      "type": ["null", "object"],
+      "additionalProperties": true
+    },
     "assignee_id": {
       "type": ["null", "integer"]
     },
+    "closed_by": {
+      "type": ["null", "object"],
+      "additionalProperties": true
```

Comment:
I was going to add all the properties into these, but many of them are only documented in examples outputs, or not at all.